docker/cloudbuild.yaml

```
@@ -8,20 +9,38 @@ steps:
   ]
   timeout: 180s
 - name: 'us.gcr.io/$PROJECT_ID/sentry-builder:$COMMIT_SHA'
+  id: builder-run
   env: [
     'SOURCE_COMMIT=$COMMIT_SHA'
   ]
   timeout: 360s
 - name: 'gcr.io/kaniko-project/executor:v0.22.0'
+  id: runtime-image-py2
+  waitFor:
+    - builder-run
   args: [
     '--cache=true',
     '--build-arg', 'SOURCE_COMMIT=$COMMIT_SHA',
     '--destination=us.gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA',
     '-f', './docker/Dockerfile'
   ]
   timeout: 300s
-# Smoke tests
+- name: 'gcr.io/kaniko-project/executor:v0.22.0'
+  id: runtime-image-py3
+  waitFor:
+    - builder-run
+  args: [
+    '--cache=true',
+    '--build-arg', 'SOURCE_COMMIT=$COMMIT_SHA',
+    '--build-arg', 'PY_VER=3.6',
```

You should match what we have in `.python-version` here.

```suggestion
    '--build-arg', 'PY_VER=3.6.10',
```

---

Why are we not using the latest patch version?

---

+1 here, let's keep this strictly pinned to match. We can just bump the patch release, but I don't fully trust docker python images without strict pinning. Things like `pip` will get upgraded or whatever in a new release that is opaque to us until it breaks, etc.